video_xblock/backends/wistia.py

```diff
@@ -168,46 +168,53 @@ def get_default_transcripts(self, **kwargs):
         # Fetch available transcripts' languages (codes and English labels), and assign its' urls.
         try:
             data = requests.get('https://' + url)
-            wistia_data = json.loads(data.text)
         except requests.exceptions.RequestException as exception:
             # Probably, current API has changed
             message = 'No timed transcript may be fetched from a video platform. ' \
                       'Error: {}'.format(str(exception))
             return default_transcripts, message
 
-        if data.status_code == status.HTTP_200_OK and wistia_data:
```

Comment:
This PR based on https://github.com/raccoongang/xblock-video/pull/137
Better look at it after merging that pr